src/components/page-seo/page-seo.tsx

```diff
@@ -0,0 +1,19 @@
+import Head from "next/head";
+import React, { useContext } from "react";
+import { ReactLocaleContext } from "../../contexts/locale-context";
+
+export interface IPageTitleProps {
+	title?: string;
```

Comment:
I think that this component's props should extend our standard page props. If you're called PageSEO, you're expected to implement SEO logic, not just render cooked data.

Reply:
Currently a custom page title is the only SEO variable we provide for pages.
We can generate more, but this means also a content job, which can be queued for a later period of time, as well as its component support.

Reply:
I'm not sure your reply addresses my concern.
I wrote that the page seo component should be responsible for collecting the data needed to O the SE. For example, it should access the metadata and collect the title and subtitle, rather than have this cooked in the containing component.

Reply:
That's exactly what the component does. It accesses `ReactLocaleContext`, picks up `title` and `subtitle` and adds a unique string for specific pages (error pages and `[id]` templates, which uses a specific meta data key).

I've decided to let the special pages cook their own title, since metadata is used there in its own turn, not just for the page title purpose.

I can think of one improvement that might address your concern, which is to move the PageSEO component to the Layout component, and not having it called everywhere. Still, this might create over-complication for the special pages (error and [id] pages).

Otherwise, I'm not understanding your comment, since this is exactly what you referred to in your original comment.

Reply:
UPDATE: I've followed my own comment from above, deprecated `PageSEO` component and move its logic to `Layout`. Now, only special pages needs to declare their static title (`error` pages) and their dynamic title (`[id]` pages), otherwise - `Layout` will fetch the correct and translated page name.